src/views/footers/local.blade.php

```diff
@@ -0,0 +1,68 @@
+<footer class="py-10 bg-gray-800">
+    <div class="px-4 text-white lg:px-16">
+        <div class="-mx-2 flex flex-wrap justify-between space-y-8 lg:space-y-0">
+            <div class="w-full px-2 lg:w-auto">
+                <a class="inline-block hover:text-gray-300 focus:outline-none focus:ring focus:ring-blue-400" href="#">
+                    {{ $logo ?? 'No logo' }}
+
+                    <h2 class="mt-1 font-black">
+                        {{ $title ?? 'No title'}}
+                    </h2>
+                </a>
+
+                <address class="mt-4 text-sm not-italic">
```

Comment:
Let's wrap this whole element in an `@isset` in case they don't want to display an address.